src/components/common/dialogs/CustomDialog.js

```diff
@@ -137,13 +139,25 @@ const CustomDialog = ({
               // https://github.com/callstack/react-native-paper/blob/main/src/components/Portal/Portal.tsx#L54
               // otherwise useContext(GlobalTogglesContext) will return undefined for
               // any custom dialog component (e.g. ExplanationDialog and other ones)
-              <PostHogProvider apiKey={Config.posthogApiKey} options={{ host: Config.posthogHost }} autocapture={false}>
```

Comment:
revert this, just keep

```jsx
<PostHogProvider apiKey={Config.posthogApiKey} options={{ host: Config.posthogHost }} autocapture={false}>
  GlobalTogglesContext.Provider value={globalToggleState}>
                   <GoodWalletContext.Provider value={goodWalletState}>
                     <DialogContext.Provider value={dialogState}>{content}</DialogContext.Provider>
                   </GoodWalletContext.Provider>
                 </GlobalTogglesContext.Provider>
</PostHogProvider>
```

as it was in my example

Reply:
I am not sure how it works but when on the WelcomeScreen apparently posthog is defined within the custom dialog, and then breaks because of the error: 'Posthog.initAsync is initialized twice'